roles/tarzan/defaults/main.yml

```diff
@@ -51,6 +51,7 @@ tarzan_env_path: "{{ luajit_dest }}/{{ luajit_version }}/bin:{{ luarocks_dest }}
 
 tarzan_log_dest: "{{ ngi_pipeline_upps_path }}/log/tarzan/"
 tarzan_db_dest: "{{ ngi_pipeline_upps_path }}/db/tarzan/"
+cassandra_log_dest: "{{ tarzan_log_dest }}/cassandra"
```

Comment:
Might want to consider moving out cassandra's log & db from /tarzan/. Depends on how hard the dependency is I suppose. Not a critique per se, just a consideration.